src/checkChange.js

```diff
@@ -0,0 +1,5 @@
+export default (li, task, listTaskClass) => {
+  const isCompleted = task.completed;
+  li.classList.toggle('completed', isCompleted);
+  listTaskClass.saveListToLocalStorage();
+};
```

Comment:
- [ ] Actually, to be honest, I couldn't understand the structure you have used in this file. I have been using `export` keyword, but you have used it in a different structure. would you please explain it to me?